airbyte-integrations/connectors/destination-iceberg-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/v2/IcebergV2CheckTest.kt

```diff
@@ -7,7 +7,9 @@ package io.airbyte.integrations.destination.iceberg.v2
 import io.airbyte.cdk.load.check.CheckIntegrationTest
 import io.airbyte.cdk.load.check.CheckTestConfig
 import io.airbyte.integrations.destination.iceberg.v2.IcebergV2TestUtil.PATH
+import org.junit.jupiter.api.Disabled
 
+@Disabled
```

Comment:
Should this still be disabled?

Reply:
Unless we dont have nessie deployed somewhere I dont think we can test it as an integration test.